### PR TITLE
Brush up additional Feat effects

### PR DESCRIPTION
**Pull request by ammalagonc**

### CODE REVIEW

File: packs/feat-effects/effect-distracting-feint.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Distracting Feint",
     "system": {
         "description": {
-            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Distracting Feint]</p>\n<p>While a creature is @UUID[Compendium.pf2e.conditionitems.Item.Off-Guard] by your Feint, it also takes a -2 circumstance penalty to Perception checks and Reflex saves.</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Distracting Feint]</p>\n<p>While a creature is Off-Guard by your Feint, it also takes a -2 circumstance penalty to Perception checks and Reflex saves.</p>"
```
Comment (simonward) [inline packs/feat-effects/effect-distracting-feint.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Distracting Feint]</p>\n<p>While a creature is off-guard by your Feint, it also takes a -2 circumstance penalty to Perception checks and Reflex saves.</p>"
```

File: packs/feat-effects/effect-hot-foot-critical-success.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Hot Foot (Critical Success)",
     "system": {
         "description": {
-            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Hot Foot]</p>\n<p>The creature is @UUID[Compendium.pf2e.conditionitems.Item.Off-Guard] and takes a -2 circumstance penalty to Reflex saves.</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Hot Foot]</p>\n<p>The creature is off-guard and takes a -2 circumstance penalty to Reflex saves.</p>"
```
Comment (simonward) [inline packs/feat-effects/effect-hot-foot-critical-success.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Hot Foot]</p>\n<p>You are off-guard and take a -2 circumstance penalty to Reflex saves.</p>"
```

File: packs/feat-effects/effect-kindle-inner-flames.json [hunk 3]
```diff
@@ -4,10 +4,10 @@
     "name": "Effect: Kindle Inner Flames",
     "system": {
         "description": {
-            "value": "<p>The creature gains a status bonus to Reflex saves and Acrobatics checks.</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Kindle Inner Flames]</p>\n<p>The creature gains a status bonus to Reflex saves and Acrobatics checks.</p>"
```
Comment (simonward) [inline packs/feat-effects/effect-kindle-inner-flames.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Kindle Inner Flames]</p>\n<p>You gain a status bonus to Reflex saves and Acrobatics checks.</p>"
```

File: packs/feat-effects/effect-leech-clip.json [hunk 4]
```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Leech-Clip",
     "system": {
         "description": {
-            "value": "<p>The target takes a –10-foot status penalty to its Speed (or a –15-foot status penalty on a critical hit).</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Leech-Clip]</p>\n<p>The target takes a –10-foot status penalty to its Speed (or a –15-foot status penalty on a critical hit).</p>"
```
Comment (simonward) [inline packs/feat-effects/effect-leech-clip.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Leech-Clip]</p>\n<p>You take a –10-foot status penalty to your Speed (or a –15-foot status penalty on a critical hit).</p>"
```

File: packs/feat-effects/effect-metallic-skin.json [hunk 5]
```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Metallic Skin",
     "system": {
         "description": {
-            "value": "<p>You fortify your natural resistances with elemental metal, covering your skin with thick metal. You gain a +2 status bonus to AC and resistance 10 to physical damage (except adamantine), but you take a –10-foot penalty to Speed. If you take fire damage while this ability is active, until the end of your next turn, you deal an additional @Damage[2d6[fire]] damage with all your unarmed melee Strikes.</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Metallic Skin]</p>\n<p>You gain a +2 status bonus to AC and resistance 10 to physical damage (except adamantine), but you take a –10-foot penalty to Speed. If you take fire damage while this ability is active, until the end of your next turn, you deal an additional @Damage[2d6[fire]] damage with all your unarmed melee Strikes.</p>"
```
Comment (simonward) [inline packs/feat-effects/effect-metallic-skin.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Metallic Skin]</p>\n<p>You gain a +2 status bonus to AC and resistance 10 to physical damage (except adamantine), but you take a –10-foot penalty to Speed. If you take fire damage while this ability is active, until the end of your next turn, you deal an additional 2d6 fire damage with all your unarmed melee Strikes.</p>"
```

File: packs/feat-effects/effect-methodical-debilitations-flanking.json [hunk 6]
```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Methodical Debilitations (Flanking)",
     "system": {
         "description": {
-            "value": "<p>The target is unable to flank or provide flanking.</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Methodical Debilitations]</p>\n<p>The target is unable to flank or provide flanking.</p>"
```
Comment (simonward) [inline packs/feat-effects/effect-methodical-debilitations-flanking.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Methodical Debilitations]</p>\n<p>You are unable to flank or provide flanking.</p>"
```

**Comment by ammalagonc** [comment]
Just in case -
Effect: Heaven's Thunder - Thinking an Item Alteration will take care of the second portion of the effect can. I'll work on this in a separate PR.

File: packs/feat-effects/effect-harrow-omen.json [hunk 7]
```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Harrow Omen",
     "system": {
         "description": {
-            "value": ""
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Harrower Dedication], @UUID[Compendium.pf2e.feats-srd.Item.Benevolent Spirit Deck], @UUID[Compendium.pf2e.feats-srd.Item.Restore Omen]</p>\n<p>Draw a card from your harrow deck—that card’s suit becomes your active harrow omen.</p>"
```
Comment (simonward) [inline packs/feat-effects/effect-harrow-omen.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Harrower Dedication], @UUID[Compendium.pf2e.feats-srd.Item.Benevolent Spirit Deck], @UUID[Compendium.pf2e.feats-srd.Item.Restore Omen]</p>\n<p>Draw a card from your harrow deck—that card's suit becomes your active harrow omen.</p>"
```

File: packs/feat-effects/effect-helpful-tinkering.json [hunk 8]
```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Helpful Tinkering",
     "system": {
         "description": {
-            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Helpful Tinkering]</p>\n<p>For 1 minute, the chosen ally's Strikes with the chosen weapon gain the same offensive boost your innovation has.</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Helpful Tinkering]</p>\n<p>Your strikes with the chosen weapon gain the same offensive boost your innovation has.</p>"
```
Comment (simonward) [inline packs/feat-effects/effect-helpful-tinkering.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Helpful Tinkering]</p>\n<p>Your strikes with the chosen weapon gain the same offensive boost your inventor ally's innovation has.</p>"
```

File: packs/feat-effects/effect-lost-in-the-crowd-10-creatures.json [hunk 9]
```diff
@@ -4,10 +4,10 @@
     "name": "Effect: Lost in the Crowd (10 Creatures)",
     "system": {
         "description": {
-            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Lost in the Crowd]</p>\n<p>You've learned how to move as one with crowds and hide among them, vanishing into a busy street in the blink of an eye. You move at full Speed in crowds and can use cover from crowds to Hide and Sneak, gaining a +2 circumstance bonus on your Stealth checks when in a crowd of at least 10 creatures.</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Lost in the Crowd]</p>\n<p>You move at full Speed in crowds and can use cover from crowds to Hide and Sneak, gaining a +2 circumstance bonus on your Stealth checks when in a crowd of at least 10 creatures.</p>"
```
Comment (simonward) [inline packs/feat-effects/effect-lost-in-the-crowd-10-creatures.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Lost in the Crowd]</p>\n<p>You gain a +2 circumstance bonus on your Stealth checks.</p>"
```

File: packs/feat-effects/effect-lost-in-the-crowd-100-creatures.json [hunk 10]
```diff
@@ -4,10 +4,10 @@
     "name": "Effect: Lost in the Crowd (100 Creatures)",
     "system": {
         "description": {
-            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Lost in the Crowd]</p>\n<p>You've learned how to move as one with crowds and hide among them, vanishing into a busy street in the blink of an eye. You move at full Speed in crowds and can use cover from crowds to Hide and Sneak, gaining a +4 circumstance bonus on your Stealth checks when in a crowd of at least 100 creatures.</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Lost in the Crowd]</p>\n<p>You move at full Speed in crowds and can use cover from crowds to Hide and Sneak, gaining a +4 circumstance bonus on your Stealth checks when in a crowd of at least 100 creatures.</p>"
```
Comment (simonward) [inline packs/feat-effects/effect-lost-in-the-crowd-100-creatures.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Lost in the Crowd]</p>\n<p>You gain a +4 circumstance bonus on your Stealth checks.</p>"
```

File: packs/feat-effects/effect-lost-in-the-crowd-10-creatures.json (hunk 9 above)
Comment (simonward) [reply packs/feat-effects/effect-lost-in-the-crowd-10-creatures.json:7]: Consider condensing the two Lost in the Crowd effects.

File: packs/feat-effects/effect-indomitable-spirit.json [hunk 11]
```diff
@@ -29,6 +29,7 @@
             },
             {
                 "key": "GrantItem",
+                "inMemoryOnly": true,
                 "onDeleteActions": {
```
Comment (simonward) [inline packs/feat-effects/effect-indomitable-spirit.json:33]:
```suggestion
```

File: packs/feat-effects/effect-indomitable-spirit.json [hunk 12]
```diff
@@ -29,6 +29,7 @@
...
                     "grantee": "restrict"
```
Comment (simonward) [inline packs/feat-effects/effect-indomitable-spirit.json:34]:
```suggestion
```

File: packs/feat-effects/effect-indomitable-spirit.json [hunk 13]
```diff
@@ -29,6 +29,7 @@
...
                 },
```
Comment (simonward) [inline packs/feat-effects/effect-indomitable-spirit.json:35]:
```suggestion
```

File: packs/feat-effects/effect-hydration.json [hunk 14]
```diff
@@ -23,7 +23,7 @@
         "rules": [
             {
                 "key": "FlatModifier",
-                "label": "Dehydrated",
+                "label": "PF2E.SpecificRule.Azarketi.Hydration.Label",
```
Comment (simonward) [inline packs/feat-effects/effect-hydration.json:26]: Maybe the effect should be renamed to "Effect: Dehydrated"? Then we wouldn't need the localized label?

File: packs/feat-effects/effect-indomitable-spirit.json [hunk 15]
```diff
@@ -29,6 +29,7 @@
             },
             {
                 "key": "GrantItem",
+                "inMemoryOnly": true,
```
Comment (simonward) [inline packs/feat-effects/effect-indomitable-spirit.json:32]: Consider sorting this key.

File: packs/feat-effects/effect-hydration.json (hunk 14 above)
Comment (ammalagonc) [reply packs/feat-effects/effect-hydration.json:26]: Yeah, let's go with that.
Makes more sense.

**Comment by TikaelSol** [comment]
Conflicts!

**Comment by ammalagonc** [comment]
Resolved

File: packs/feat-effects/effect-dehydrated.json [hunk 16]
```diff
@@ -1,10 +1,10 @@
 {
     "_id": "Q5FUu7yhWPJlcXei",
     "img": "icons/magic/defensive/armor-stone-skin.webp",
-    "name": "Effect: Hydration",
+    "name": "Effect: Dehydrated",
     "system": {
         "description": {
-            "value": "<p>Granted by @UUID[Compendium.pf2e.ancestryfeatures.Item.Hydration] if you spend 24 hours outside of water.</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.ancestryfeatures.Item.Hydration]</p>\n<p>You take a -1 status penalty to Fortitude saves.</p>"
```
Comment (simonward) [inline packs/feat-effects/effect-dehydrated.json:7]: The FlatModifier label can be removed.

File: static/lang/re-en.json [hunk 17]
```diff
@@ -3986,6 +3986,13 @@
                     "Note": "Each of your successful saving throws against a disease reduces its stage by 2, or by 1 for a virulent disease. Each critical success against an ongoing disease reduces its stage by 3, or by 2 for a virulent disease."
                 }
             },
+            "LostInTheCrowd": {
+              "CrowdSize": {
+                "Ten": "At least 10 creatures",
+                "Hundred": "At least 100 creatures"
+              },
+              "Prompt": "Crowd Size"
```
Comment (simonward) [inline static/lang/re-en.json:3994]:
```suggestion
              "Prompt": "Select a crowd size."  
```

**Pull request merged**